aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker/samples/2.0/HelloDockerTools/Pages/Index.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home page!!!!!";
```

Comment:
Thank you. :+1:

Reply:
We need to have a little fun with the samples, right?

Reply:
🤦‍♂️ I need to *eat my remark now*. I just discovered in a Blazor sample that I told the reader to add exclamation points to a heading in a sample app ... Do what I *say*, not what I *do*. That's the advice I follow ..... I only do what I *say* .... I never *do* what I *do*. :smile:

![39509668-e8949a92-4d9b-11e8-91e9-b6a494695d92](https://user-images.githubusercontent.com/1622880/43284376-310984f8-90e1-11e8-8862-5d190bfe1514.png)